addons/account/tests/test_reconcile_account.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+from odoo.addons.account.tests.test_reconciliation import TestReconciliation
+
+
+class TestReconcileBlueLinesRestriction(TestReconciliation):
```

Comment:
Your branch is red. I'd say you're missing @tagged('post_install', '-at_install') before this line ;)

Reply:
Thanks for the pointer :wink: